pkg/uploader/types.go

```diff
@@ -39,12 +39,17 @@ const (
 
 // ValidateUploaderType validates if the input param is a valid uploader type.
 // It will return an error if it's invalid.
-func ValidateUploaderType(t string) error {
+func ValidateUploaderType(t string) (string, error) {
 	t = strings.TrimSpace(t)
 	if t != ResticType && t != KopiaType {
-		return fmt.Errorf("invalid uploader type '%s', valid upload types are: '%s', '%s'", t, ResticType, KopiaType)
+		return "", fmt.Errorf("invalid uploader type '%s', valid upload types are: '%s', '%s'", t, ResticType, KopiaType)
 	}
-	return nil
+
+	if t == ResticType {
+		return fmt.Sprintf("Uploader '%s' is deprecated, don't use it for new backups, otherwise, the backed up data may not be accessed by new versions of Velero", t), nil
```

Comment:
@sseago any feedback for the wording?

Reply:
@reasonerjt I'd replace "otherwise, the backed up data may not be accessed by new versions of Velero" with "otherwise this backup won't be available for restore when this functionality is removed in a future version of Velero"

Reply:
Done, thanks for the suggestion @sseago